src/js/media-store/request-map.ts

```diff
@@ -161,10 +161,13 @@ export const requestMap: RequestMap = {
   [MediaUIEvents.MEDIA_UNMUTE_REQUEST](stateMediator, stateOwners) {
     const key = 'mediaMuted';
     const value = false;
-    // If we've unmuted but our volume is currently 0, automatically set it to some low volume
-    if (!stateMediator.mediaVolume.get(stateOwners)) {
-      stateMediator.mediaVolume.set(0.25, stateOwners);
-    }
+    const volumePref =
+      +globalThis.localStorage.getItem('media-chrome-pref-volume') || +'0.25';
```

Comment:
```suggestion
      +globalThis.localStorage.getItem('media-chrome-pref-volume');
```

that fallback doesn't seem needed because below you're already doing a check
`NaN > 0.25 = false`